src/langs/en/messages.json

```
@@ -119,5 +119,6 @@
   "DELETE_SHORT_LINK": "The short link was successfully deleted.",
   "SHORT_LINK_ALREADY_EXISTS": "The short link's alias already exists.",
   "SHORT_LINK_NOT_FOUND": "The short link was not found.",
-  "INVALID_CONTEXT": "The given context is not valid."
+  "INVALID_CONTEXT": "The given context is not valid.",
+  "SHORT_LINK_LIMIT_EXCEED": "The limit of short links exceeded. You can't have more than one short link per context."
```

Not sure If we should use Platform or Context ? Maybe we should call it "service" ?
```suggestion
  "SHORT_LINK_LIMIT_EXCEED": "You can't have more than one short link per platform."
```

---

Yes, I think that's a good idea because service would be clearer to users than platform or context.